OpenNefia.Content/TurnOrder/TurnOrderSystem.cs

```diff
@@ -42,6 +42,11 @@ public interface ITurnOrderSystem : IEntitySystem
         /// Advances the state of the simulation until additional input or exiting are needed.
         /// </summary>
         void AdvanceStateFromPlayer(TurnResult turnResult);
+
```

Comment:
Can this be moved into its own system, and hook into `PlayerTurnStartedEvent`? This system is only concerned with the core turn order logic.

Reply:
changed in 5da0180